substrate/frame/system/src/lib.rs

```diff
@@ -691,7 +689,6 @@ pub mod pallet {
 			<UpgradedToU32RefCount<T>>::put(true);
 			<UpgradedToTripleRefCount<T>>::put(true);
 
-			sp_io::storage::set(well_known_keys::CODE, &self.code);
 			sp_io::storage::set(well_known_keys::EXTRINSIC_INDEX, &0u32.encode());
```

Comment:
Looking at this now I wonder what is the point of setting to 0 the well known EXTRINSIC_INDEX during build.

 Maybe we can modify [this](https://github.com/paritytech/substrate/blob/2e7932fac159aa8855feeed7578319ae0e6d8296/frame/system/src/lib.rs#L1321-L1323) with an unwrap_or_default() (thus not returning an Option).
 
 On block initialization this is [always set to 0](https://github.com/paritytech/substrate/blob/2e7932fac159aa8855feeed7578319ae0e6d8296/frame/system/src/lib.rs#L1359)

Reply:
I'd prefer to not touch it in this PR. It is already bloaty.